docs/manual/en/Makefile

```diff
@@ -0,0 +1,13 @@
+
+ADOC = asciidoctor
+ADOCFLAGS = -a docdate="$(shell date "+%d/%m/%Y")" -a doctime="$(shell date "+%R %Z")"
```

Comment:
I think english can do without these flags, defaults would be fine, these being overrides for time and date formatting.